src/streams/tests/test_libaeon_streams/test_stream_reader.cpp

```diff
@@ -35,7 +35,7 @@
 struct test_fixture_stream_reader_default_data : public ::testing::Test
 {
     test_fixture_stream_reader_default_data()
-        : fixture_data({{'A', 'B', 'C', 'D', 'E', '\n', '1', '2', '3', '4'}})
+        : fixture_data({{'A', 'B', 'C', 'D', 'E', '\r', '\n', '1', '2', '3', '4'}})
```

Comment:
The point was, that '\r' wasn't tested. By adding '\r' to this test string, both '\r' and '\n' behavior is tested.